spec/requests/api/allocations_controller_create_spec.rb

```diff
@@ -255,74 +232,7 @@
         ]
       end
 
-      it_behaves_like 'an endpoint that responds with error 422'
-
-      it 'does not create associated moves' do
-        expect { post '/api/v1/allocations', params: { data: data }, headers: headers, as: :json }
-          .not_to change(Move, :count)
-      end
-
-      context 'when the supplier has a webhook subscription', :skip_before do
-        let!(:subscription) { create(:subscription, :no_email_address, supplier: supplier) }
-        let!(:notification_type_webhook) { create(:notification_type, :webhook) }
-
-        before do
-          perform_enqueued_jobs(only: [PrepareMoveNotificationsJob, NotifyWebhookJob]) do
-            post '/api/v1/allocations', params: { data: data }, headers: headers, as: :json
-          end
-        end
-
-        it 'does not enqueue notifications' do
-          expect(subscription.notifications.count).to be_zero
-        end
-      end
-
-      context 'when the supplier has an email subscription', :skip_before do
-        let!(:subscription) { create(:subscription, :no_callback_url, supplier: supplier) }
-        let!(:notification_type_email) { create(:notification_type, :email) }
-
-        before do
-          perform_enqueued_jobs(only: [PrepareMoveNotificationsJob, NotifyEmailJob]) do
-            post '/api/v1/allocations', params: { data: data }, headers: headers, as: :json
-          end
-        end
-
-        it 'does not enqueue notifications' do
-          expect(subscription.notifications.count).to be_zero
-        end
-      end
-    end
-
-    context 'with incorrect moves_count type' do
```

Comment:
an invalid moves_count would be good test for - either here or in a params validator

Reply:
ignore ^ , I've found the test 👍